### DISCUSSION TITLE
Delete link folder in repositories

### DISCUSSION THREAD

**Discussion by USeebi** [community/community#142489] [Repositories]
### Select Topic Area

Question

### Body

A new repository is created from git import.  The original repository contain a folder link which I don't need.  Tried to delete it in github.dev and the link removed but Source Control reported nothing change in commit.

The link folder reappear in github.com and github.dev. What I need to do to remove the folder link?
Thanks for any pointer!

**Comment by MasterInQuestion** [comment]
You mean "git submodules"?
    I fear they're only operatable from &#x60;git&#x60;...

    Relevant research:
    https://github.com/MasterInQuestion/TEST2

**Reply by USeebi** [reply]
Thanks, I got the submodule deleted on Github Desktop
